tests/features/service-selector.feature

```diff
@@ -0,0 +1,80 @@
+
+
+Scenario: Call setItemsStream with stream of items when Selector instance is empty
+    Given Selector Service with setItemsStream method
+    When  user calls setItemsStream with a stream of items
+    Then  all items are being loaded in the Selector instance
```

Comment:
`Then  all items are being loaded in the Selector instance` this step is not needed, this is not something that will be checked.